.travis.yml

```diff
@@ -15,8 +15,13 @@ os:
 
 matrix:
   include:
-    - os: osx
-      rust: stable
+  - os: osx
+    rust: stable
+  - rust: nightly-2018-04-06
```

Comment:
This version is used by TiKV but `rust-rocksdb` doesn't have this same requirement. I am worried later when we upgrade the version used by TiKV it means we need to change this. Perhaps it is better to do it on `stable`?

Reply:
OK, let me try it on stable.